packages/functional-tests/tests/react-conversion/cannotCreateAccount.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { test, expect } from '../../lib/fixtures/standard';
```

Comment:
Could add a license here. Also do we want to add those experiment params anywhere (even in a quick README in the `react-conversion/` directory)? I'm sure we'll figure it out when we get there too.